### DISCUSSION TITLE
redirect to already existing rtsp

### DISCUSSION THREAD

**Discussion by andmancio** [bluenviron/mediamtx#383] [Questions]
hi i have a stream at url rtsp://10.202.0.1/live (:554) and i want to be accessible from my device ip 192.168.1.11, i use redirected option like you described:
`paths:
  redirected:
    # source of the stream - this can be:
    # * record -> the stream is published by a RTSP or RTMP client
    # * rtsp://existing-url -> the stream is pulled from another RTSP server
    # * rtsps://existing-url -> the stream is pulled from another RTSP server, with RTSPS
    # * rtmp://existing-url -> the stream is pulled from a RTMP server
    # * redirect -> the stream is provided by another path or server
    source: redirect

    # if the source is an RTSP or RTSPS URL, this is the protocol that will be used to
    # pull the stream. available options are "automatic", "udp", "tcp".
    # the tcp protocol can help to overcome the error "no UDP packets received recently".
    sourceProtocol: automatic

    # if the source is "redirect", this is the RTSP URL which clients will be
    # redirected to.
    sourceRedirect: rtsp://10.202.0.1/live`

but when i try to play it with vlc at address rtsp://192.168.1.11:8554 doesn't work.
any suggestion?
many thanks

**Comment by aler9** [comment]
Hello, you have to use the `source` parameter like this

```yml
paths:
  mypath:
    source: rtsp://10.202.0.1/live
```

and the use this URL in VLC

```
rtsp://192.168.1.11:8554/mypath
```

**Reply by andmancio** [reply]
Thank you!! i will try it!!

**Reply by andmancio** [reply]
@aler9 i tried but i received this error:
[0/0] rtsp-simple-server v0.16.0
2021/05/29 18:12:45 I [0/0] [RTSP] UDP/RTP listener opened on :8000
2021/05/29 18:12:45 I [0/0] [RTSP] UDP/RTCP listener opened on :8001
2021/05/29 18:12:45 I [0/0] [RTSP] TCP listener opened on :8554
2021/05/29 18:12:45 I [0/0] [RTMP] listener opened on :1935
2021/05/29 18:12:45 I [0/0] [HLS] listener opened on :8888
2021/05/29 18:12:45 I [0/0] [path mypath] [rtsp source] started
2021/05/29 18:12:45 I [0/0] [path mypath] [rtsp source] ERR: wrong status code: 501 (Not Implemented)

can you help me?
Thanks

**Reply by aler9** [reply]
Hello, this could be a compatibility issue between the server and the camera. Please open an issue by following the instructions and post a full log with `logLevel: debug` in the configuration file.

**Reply by andmancio** [reply]
ok thank you!!

[a comment by an automated account is not shown]
